packages/rules-typescript/src/src-detector.ts

```diff
@@ -50,11 +50,17 @@ export class SrcDetector implements PackageRule {
 
   getImportPaths(sourceFile: SourceFile): ImportPathAndLineNumber[] {
     const importPaths: ImportPathAndLineNumber[] = [];
+    let lineNumber = 1;
     sourceFile.forEachChild(n => {
+      const totalLines = n.getFullText().split(/\r?\n/).length;
+      lineNumber = lineNumber + totalLines - 1;
       if (isImportDeclaration(n)) {
+        console.log("==================================>>>>>>>>>>>>");
+        console.log("path : " + n.moduleSpecifier.getText());
```

Comment:
Did you mean to remove these console logs?

Reply:
Yes. I removed it.